src/Stratis.Bitcoin.Features.PoA/Voting/VotingManager.cs

```diff
@@ -429,6 +429,14 @@ private void OnBlockConnected(BlockConnected blockConnected)
                         {
                             ChainedHeader chainedHeader = chBlock.ChainedHeader.GetAncestor(poll.PollStartBlockData.Height);
 
+                            if (chainedHeader?.Header == null)
+                            {
+                                this.logger.LogDebug("Couldn't retrieve header for block at height: {0}.", poll.PollStartBlockData?.Height);
```

Comment:
Please change to LogWarning and add the hash as well..

Reply:
Done.